osu.Game/Graphics/Containers/SectionsContainer.cs

```diff
@@ -150,31 +158,63 @@ public override void Add(T drawable)
             footerHeight = null;
         }
 
+        private ScheduledDelegate? scrollToTargetDelegate;
+
         public void ScrollTo(Drawable target)
         {
-            lastKnownScroll = null;
+            Logger.Log($"Scrolling to {target}..");
```

Comment:
Are the log statements in this class left here intentionally, or is this a debugging leftover? These still being in here feel a bit off to me.

Reply:
It's a pretty rare operation so I figured I'd leave them there until they get annoying. At least then if this goes wrong in the wild we have something to work with.

Can remove if you'd prefer.

Reply:
Let's go with it for now.